bitnami/grafana/templates/deployment.yaml

```
@@ -113,10 +113,16 @@ spec:
               mountPath: /opt/bitnami/grafana/conf/provisioning/dashboards
             {{- end }}
             {{- range .Values.dashboardsConfigMaps }}
+            {{- if .folderName }}
+            - name: {{ .configMapName }}
+              mountPath: /opt/bitnami/grafana/dashboards/{{ .folderName }}/{{ .fileName }}
+              subPath: {{ .fileName }}
+            {{- else }}
             - name: {{ .configMapName }}
               mountPath: /opt/bitnami/grafana/dashboards/{{ .fileName }}
               subPath: {{ .fileName }}
             {{- end }}
```

name and subPath are the same in both cases, you can define them outside the if-else statement:
```suggestion
            - name: {{ .configMapName }}
            {{- if .folderName }}
              mountPath: /opt/bitnami/grafana/dashboards/{{ .folderName }}/{{ .fileName }}
            {{- else }}
              mountPath: /opt/bitnami/grafana/dashboards/{{ .fileName }}
            {{- end }}
              subPath: {{ .fileName }}
```